Two95JavaTraining/Arun/programming/assignments/Duplicate.java

```diff
@@ -0,0 +1,37 @@
+package programming.assignments;
+
+public class Duplicate {

```

Comment:
always provide comments on top of the class..what the class is supposed to do? author etc?